prereise/gather/demanddata/transportation_electrification/data_helper.py

```diff
@@ -89,3 +90,56 @@ def update_if_weekend(data: pd.DataFrame):
     data.loc[data["Day of Week"].isin(range(2, 7)), "If Weekend"] = 2
 
     return data
+
+
+def get_total_daily_vmt(
+    data: pd.DataFrame,
+    comm_type: int,
+    locationstrategy: List[int],
+    input_day: List[int],
+    data_day: np.array,
+):
+    """Load data and use the parameters to calculate total_daily_vmt.
```

Comment:
What do we calculate exactly? Can we be more explicit here.

Reply:
more specific now